nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/state/repo/DefaultModelRepositoryFactory.java

```diff
@@ -53,5 +52,13 @@ public ModelRepository get(Repo.Database db) {
 	public ModelPersistenceHandler getHandler(Repo repo) {
 		return getBeanResolver().get(ModelPersistenceHandler.class, repo.value().name()+"_handler");
 	}
+
+	@Override
+	public ModelRepository get(Repo repo, ModelConfig<?> mConfig) {
```

Comment:
Pls refer to comment provided earlier in cases when this method was getting invoked.